catalog/ui/catalog-ui-search/src/test/java/org/codice/ddf/catalog/ui/query/suggestion/UtmUpsCoordinateProcessorTest.java

```diff
@@ -14,65 +14,46 @@
 package org.codice.ddf.catalog.ui.query.suggestion;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.closeTo;
 import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.hasSize;
-import static org.hamcrest.Matchers.not;
 import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsEqual.equalTo;
-import static org.mockito.Mockito.doReturn;
-import static org.mockito.Mockito.mock;
 
-import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
 import java.util.LinkedList;
 import java.util.List;
 import org.codice.ddf.spatial.geocoding.Suggestion;
 import org.codice.usng4j.CoordinateSystemTranslator;
-import org.codice.usng4j.DecimalDegreesCoordinate;
-import org.codice.usng4j.UtmUpsCoordinate;
-import org.junit.Before;
+import org.codice.usng4j.impl.CoordinateSystemTranslatorImpl;
 import org.junit.Test;
 
 public class UtmUpsCoordinateProcessorTest {
```

Comment:
As a sanity check, I ran these tests against master without my other changes and all the tests passed except `testUtmStringWithoutLatBand`, `testUtmStringWithNLatitudeBand`, and `testUtmStringWithNLatitudeBand`, which is expected